tap-client/src/components/footer/FooterComponent.tsx

```diff
@@ -1,3 +1,61 @@
-export default function FormComponent() {
-    return "";
+export default function FooterComponent() {
+    return (
+        <footer className="bg-towsonBlack text-towsonWhite border-t-4 border-towsonGold">
+            {/* Links Section */}
+            <div className="flex justify-center space-x-8 py-6 text-lg">
+                <a
+                    href="/home"
+                    className="hover:text-towsonLineGold transition duration-300"
+                >
+                    Home
+                </a>
+                <a
+                    href="/about"
+                    className="hover:text-towsonLineGold transition duration-300"
+                >
+                    About
+                </a>
+                <a
+                    href="/faq"
+                    className="hover:text-towsonLineGold transition duration-300"
+                >
+                    FAQ
+                </a>
+                <a
+                    href="/form"
+                    className="hover:text-towsonLineGold transition duration-300"
+                >
+                    Form
+                </a>
```

Comment:
Remove Form with Contact Us. Just set it the href to /contact-us. I'll deal with this component later.

Reply:
Re-request me after you complete it